odoo/addons/base/models/res_partner.py

```diff
@@ -418,6 +418,12 @@ def _compute_company_registry(self):
         for company in self:
             company.company_registry = company.company_registry
 
+    @api.model
+    def _get_view_cache_key(self, view_id=None, view_type='form', **options):
+        """Add context variable force_email in the key as _get_view depends on it."""
+        key = super()._get_view_cache_key(view_id, view_type, **options)
+        return key + (self._context.get('force_email'),)
```

Comment:
This is a change for mail and not for base.
Please move this override in mail addon. Thanks

Reply:
Yes, you're right, sorry. I will update the code and move that override in mail.